po/update_pot.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+SCRIPT_NAME=$(basename "$0")
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [option]"
+
+Update POT files from codebase and production database.
+
+Option:
+  --commit  Commit changes to Git, if any
+EOH
+)
+
+if [ $# -gt 1 ]
+then
+  echo >&2 "$SCRIPT_NAME: too many arguments"
+  echo >&2 "$HELP"
+  exit 64
+elif [ $# -eq 1 ]
+then
+  if [ "$1" != '--commit' ]
+  then
+    echo >&2 "$SCRIPT_NAME: unrecognized option: $1"
+    echo >&2 "$HELP"
+    exit 64
+  fi
+  DO_COMMIT='yes'
+else
+  DO_COMMIT='no'
+fi
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+
+cd "$MB_SERVER_ROOT"
+
+if [ $DO_COMMIT = 'yes' ] && \
+  ! (git diff --quiet && git diff --cached --quiet)
+then
+  sed -E 's/^ *> ?//' >&2 << ..EOM
+  > $SCRIPT_NAME: Git working tree has local changes already
+  >
+  > Your local changes would be incidentally committed with POT files.
+  > Please clean your Git working tree before updating POT files.
+..EOM
+  exit 70
+fi
+
+export MB_POT_DB=PROD_STANDBY
+
+if ! script/database_exists $MB_POT_DB
+then
+  exit 78
+fi
+
+cd po
+
+touch extract_pot_db extract_pot_templates
+make pot
+
+if [ $DO_COMMIT = 'yes' ]
+then
+  if git diff --quiet
+  then
+    echo "POT files already up-to-date, nothing to commit."
+  else
+    git add -- *.po
+    git commit -m 'Update POT files using code and production database'
```

Comment:
With "current" removed I feel like "using code" isn't actually saying anything meaningful, so maybe just "Update POT files using the production database" is fine? Bonus: it's 46 chars.

Reply:
I initially wanted to replace “current” with actual branch/commit but somehow lost that change.

Reply:
Anyway, it would not make sense to write “patching current code” for each commit either.

Retained your 46 chars in https://github.com/metabrainz/musicbrainz-server/pull/1337/commits/eefca85dd657a6a32e61f84f2f3c11a1364dfd11